python/dex/native_function.py

```diff
@@ -19,6 +19,20 @@
 ]
 IdxRepTy = ctypes.c_uint32
 
+
+def dex_annotation(ctype):
+  dex_types = {
+      ctypes.c_int64: 'Int64',
+      ctypes.c_int32: 'Int32',
+      ctypes.c_uint8: 'Word8',
+      ctypes.c_uint32: 'Word32',
+      ctypes.c_uint64: 'Word64',
+      ctypes.c_double: 'Float64',
+      ctypes.c_float: 'Float32',
+  }
```

Comment:
This creates the map from scratch every time you call the function, only to look up one of the elements. Perhaps just make the map global? and inline the expression since then the remaining body is so trivial?

Reply:
Done.